djangocms_concurrent_users/static/js/concurrent_users.js

```diff
@@ -14,7 +15,20 @@ var ConcurrentUsers = {
     },
     hideOverlay: function hideOverlay() {
         $('.js-preventClickOverlay').hide();
-        $('.js-concurrentWarning').remove();
+        $('.js-concurrentWarning').hide();
+    },
+    showMessage: function(data) {
+        // var message_bar = '<div class="js-concurrentWarning concurrentWarning">' + data.message + '</div>'
+        // $('body, .cms-structure').append(message_bar);
+        $('.js-concurrentWarningMessage').text(data.message);
+        $('.js-concurrentWarningButtonPublishedPage').attr('href', (data.buttons.published_page.link));
+        $('.js-concurrentWarningButtonPublishedPage').text(data.buttons.published_page.link_text);
+        $('.js-concurrentWarningButtonBack').attr('href', (data.buttons.back.link));
+        $('.js-concurrentWarningButtonBack').text(data.buttons.back.link_text);
```

Comment:
In terms of style and performance: I would suggest, that you remove the duplicated selectors. So instead of 
``$('.js-concurrentWarningButtonBack').attr('href', (data.buttons.back.link)); ``
``$('.js-concurrentWarningButtonBack').text(data.buttons.back.link_text);`` 
you should write 
``$('.js-concurrentWarningButtonBack').attr('href', (data.buttons.back.link))``
``.text(data.buttons.back.link_text);``